vars/spark.yaml

```diff
@@ -10,3 +10,13 @@ AWS_JAVA_SDK_CORE_URL: https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk
 AWS_JAVA_SDK_DYMODB_URL: https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-dynamodb/1.11.687/aws-java-sdk-dynamodb-1.11.687.jar
 AWS_JAVA_SDK_S3_URL: https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-s3/1.11.687/aws-java-sdk-s3-1.11.687.jar
 GCS_CONNECTOR_HADOOP_URL: https://storage.googleapis.com/hadoop-lib/gcs/gcs-connector-hadoop3-latest.jar
+# Version specific files
+CUDA_TOOLKIT_UBUNTU_18.04_10.1_URL: http://developer.download.nvidia.com/compute/cuda/10.1/Prod/local_installers/cuda_10.1.243_418.87.00_linux.run
+CUDA_TOOLKIT_UBUNTU_18.04_10.2_URL: http://developer.download.nvidia.com/compute/cuda/10.2/Prod/local_installers/cuda_10.2.89_440.33.01_linux.run
+CUDA_TOOLKIT_UBUNTU_18.04_11.0_URL: https://developer.download.nvidia.com/compute/cuda/11.0.3/local_installers/cuda_11.0.3_450.51.06_linux.run
+CUDA_TOOLKIT_UBUNTU_18.04_11.1_URL: https://developer.download.nvidia.com/compute/cuda/11.1.0/local_installers/cuda_11.1.0_455.23.05_linux.run
+CUDF_14_10.1_URL: https://repo1.maven.org/maven2/ai/rapids/cudf/0.14/cudf-0.14-cuda10-1.jar
+CUDF_14_10.2_URL: https://repo1.maven.org/maven2/ai/rapids/cudf/0.14/cudf-0.14-cuda10-2.jar
+CUDF_15_10.1_URL: https://repo.maven.apache.org/maven2/ai/rapids/cudf/0.15/cudf-0.15-cuda10-1.jar
+CUDF_15_10.2_URL: https://repo.maven.apache.org/maven2/ai/rapids/cudf/0.15/cudf-0.15-cuda10-2.jar
+CUDF_15_11.0_URL: https://repo.maven.apache.org/maven2/ai/rapids/cudf/0.15/cudf-0.15-cuda11.jar
```

Comment:
Where are you calling these packages to be downloaded and placed?

Reply:
I am not calling them to be downloaded. I am creating variables so we can ultimately improve the scripts to do a specific version of the driver installation and include the appropriate files.